.github/workflows/ci.yml

```diff
@@ -9,19 +9,6 @@ on:
     branches:
       - master
```

Comment:
Do these changes bring the Linkerd adapter repo inline with the ci.yml of the other adapter repos (verbatim)?

Reply:
I do not clear what your mean

Reply:
We use `golangci-lint` includes linter `golint`.  So, we do not need to add another one.